src/test/java/uk/gov/dhsc/htbhf/page/BasePage.java

```diff
@@ -119,6 +123,11 @@ public boolean isBackLinkPresent() {
         return !backLink.isEmpty();
     }
 
+    public void executeSetNoValidateScript() {
```

Comment:
Might be worth some javadoc explaining that this switches off in-browser validation of fields such as email addresses

Reply:
Ah yes, this is in the enter email section so I'll add it here too for clarity.